mappings/net/minecraft/util/registry/Registry.mapping

```diff
@@ -148,3 +151,12 @@ CLASS net/minecraft/class_2378 net/minecraft/util/registry/Registry
 		ARG 1 value
 	METHOD method_29722 getEntries ()Ljava/util/Set;
 	METHOD method_30517 getKey ()Lnet/minecraft/class_5321;
+	METHOD method_31138 getLifecycle ()Lcom/mojang/serialization/Lifecycle;
+	METHOD method_31139 getEntryLifecycle (Ljava/lang/Object;)Lcom/mojang/serialization/Lifecycle;
+	METHOD method_31140 require (Lnet/minecraft/class_5321;)Ljava/lang/Object;
```

Comment:
Really need a better name for this.

Reply:
getRequired?

Reply:
That makes me think of a method which gets a registry entry which is marked as required.

This is basically `get(RegistryKey<T>)` but non-null and throws if the value is not present

Reply:
Apparently cannot think of a better name than `require` unfortunately. I honestly would rename the optional one to `find` but that's out of scope. This is good.

Reply:
```suggestion
	METHOD method_31140 getRequired (Lnet/minecraft/class_5321;)Ljava/lang/Object;
```
I think `getRequired` is still better than `require`

Reply:
`getOrExcept`/`getOrThrow`?

Reply:
getRequired makes me think it has to be a builtin biome to use the method. getOrThrow could work but is not as succinct as `require`

Reply:
`require` is better than `getRequired`